src/PrestaShopBundle/Resources/views/Admin/Sell/Customer/Blocks/form.html.twig

```diff
@@ -0,0 +1,217 @@
+{#**
+  * 2007-2018 PrestaShop
+  *
+  * NOTICE OF LICENSE
+  *
+  * This source file is subject to the Open Software License (OSL 3.0)
+  * that is bundled with this package in the file LICENSE.txt.
+  * It is also available through the world-wide-web at this URL:
+  * https://opensource.org/licenses/OSL-3.0
+  * If you did not receive a copy of the license and are unable to
+  * obtain it through the world-wide-web, please send an email
+  * to license@prestashop.com so we can send you a copy immediately.
+  *
+  * DISCLAIMER
+  *
+  * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+  * versions in the future. If you wish to customize PrestaShop for your
+  * needs please refer to http://www.prestashop.com for more information.
+  *
+  * @author    PrestaShop SA <contact@prestashop.com>
+  * @copyright 2007-2018 PrestaShop SA
+  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+  * International Registered Trademark & Property of PrestaShop SA
+  *#}
+
+{% set isGuest = isGuest|default(false) %}
+
+{% block customer_form %}
+  {{ form_start(customerForm) }}
+    <div class="card">
+      <h3 class="card-header">
+        <i class="material-icons">person</i>
+        {{ 'Customer'|trans({}, 'Admin.Global') }}
+      </h3>
+      <div class="card-block row">
+        <div class="card-text">
+          <div class="form-group row">
+            <label for="{{ customerForm.gender_id.vars.id }}" class="form-control-label">{{ 'Social title'|trans({}, 'Admin.Global') }}</label>
+            <div class="col-sm">
+              {{ form_widget(customerForm.gender_id) }}
+            </div>
+          </div>
+
+          <div class="form-group row">
+            <label for="{{ customerForm.first_name.vars.id }}" class="form-control-label">
+              <span class="text-danger">*</span>
+              {{ 'First name'|trans({}, 'Admin.Global') }}
+            </label>
+            <div class="col-sm">
+              {{ form_widget(customerForm.first_name) }}
+              <small class="form-text">
+                {{ '%s %s'|format('Invalid characters:'|trans({}, 'Admin.Notifications.Info'), '0-9!<>,;?=+()@#"°{}_$%:') }}
```

Comment:
This placeholder / regexp is duplicated 4 times in this PR (see https://github.com/PrestaShop/PrestaShop/pull/11723/files#diff-010164e39ae7060d027789c56c50876cR72). I think we can refactor that ;)

Reply:
yup, actually `FristName` and `LastName` have same constraint, but even though they happen to have same values, they are totally different constraints and in theory, if one of those rules changes, we should be able to change any of those 2 without affecting each other.

but maybe i understand it in a wrong way, wdyt? :thinking:

Reply:
Ok to keep 2 regexps separated, one for `FirstName`, one for `LastName`. But can we avoid the duplication of the regexp between the VO and the twig template ?